docs/web_reference.rst

```diff
@@ -2747,8 +2747,8 @@ Utilities
 
 .. class:: FileField
 
-   A :class:`~collections.namedtuple` instance that is returned as
-   multidict value by :meth:`Request.POST` if field is uploaded file.
+   A :class:`~typing.NamedTuple` instance that is returned as
```

Comment:
This is actually a dataclass on master https://github.com/aio-libs/aiohttp/blob/master/aiohttp/web_request.py#L74 and an `attr.s` class on the 3.8 branch.

Reply:
Forget about this, data or attr.s?

Reply:
Not data. Dataclass in master and attrs in 3.8.

Reply:
```suggestion
   A :mod:`dataclass <dataclasses>` instance that is returned as
```